tests/image/Dockerfile

```diff
@@ -0,0 +1,145 @@
+# docker build -t registry.gitlab.com/altinity-qa/clickhouse/cicd/clickhouse-operator .
+FROM ubuntu:20.04
+
+RUN apt-get update \
+    && env DEBIAN_FRONTEND=noninteractive apt-get install --yes \
+    apt-transport-https \
+    software-properties-common \
+    htop \
+    vim \
+    gettext-base \
+    ethtool \
+    jq \
+    socat \
+    make \
+    gcc \
+    g++ \
+    gpg \
+    ca-certificates \
+    bash \
+    btrfs-progs \
+    e2fsprogs \
+    iptables \
+    xfsprogs \
+    tar \
+    pigz \
+    wget \
+    git \
+    perl \
+    iproute2 \
+    cgroupfs-mount \
+    python3-pip \
+    tzdata \
+    psmisc \
+    libreadline-dev \
+    libicu-dev \
+    bsdutils \
+    curl \
+    liblua5.1-dev \
+    luajit \
+    libssl-dev \
+    libcurl4-openssl-dev \
+    gdb \
+    && rm -rf \
+        /var/lib/apt/lists/* \
+        /var/cache/debconf \
+        /tmp/* \
+    && apt-get clean
+
+
+RUN curl -fsSL https://download.docker.com/linux/ubuntu/gpg | gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg
+RUN echo "deb [arch=amd64 signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" | tee /etc/apt/sources.list.d/docker.list > /dev/null
+RUN apt-get update
+RUN apt-get install --yes docker-ce docker-ce-cli containerd.io
+
+RUN curl -fsSLo /usr/share/keyrings/kubernetes-archive-keyring.gpg https://packages.cloud.google.com/apt/doc/apt-key.gpg
+RUN echo "deb [signed-by=/usr/share/keyrings/kubernetes-archive-keyring.gpg] https://apt.kubernetes.io/ kubernetes-xenial main" | tee /etc/apt/sources.list.d/kubernetes.list
+RUN curl -LO https://storage.googleapis.com/minikube/releases/v1.23.0/minikube-linux-amd64
+RUN install minikube-linux-amd64 /usr/local/bin/minikube
+RUN apt-get update
+RUN apt-get install -y kubectl
+
+ENV TZ=Europe/Moscow
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+RUN pip3 install urllib3 testflows==1.7.4 PyYAML docker-compose==1.29.1 docker==5.0.0 dicttoxml tzlocal python-dateutil setuptools
```

Comment:
could you move python requirements to `./tests/requirements.txt`

Reply:
No, these are packages installen INSIDE docker image, not in bare system

Reply:
What's stopping you from 
```
COPY requirements.txt
RUN pip3 install -U -r requirements.txt
```
?

Reply:
And what's the point? It will be the same, except we'll need one more COPY

Reply:
`requirements.txt` it's a standard way provided by python
currently, when you need update requirements, you will do it twice
one time in `requirements.txt`, second time in your Dockerfile

@vzakaznikov jfyi